docker/bin/await-service

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Wait for a service to accept connections.
```

Comment:
Good point. I see Etcd has a /health endpoint for that purpose.
Do the other components have one we should use too?